config/settings.yml

```diff
@@ -16,6 +16,9 @@ relative_url_root: /
 
 secret_key_base: 8af0fe1e378586520e4324694897eb269bd0fffa1c5be6cc3b4ffb9dbde095d0bef5c7fdab73cd05685d8fe1dd589287d78b38e4de7116fbe14461e414072677
 
+set_sso_cookie: true
```

Comment:
Should this be `false` by default and set to true in Staging? Or are we okay with having this in Prod?

Reply:
I think this should probably be false. According to conversations with James, we can push a configuration for staging and those only take a few minutes to propogate vs a complete build which takes 15-20 minutes.

Thanks @wyattwalter for calling attention to this.

@dhall your specs will likely break with this change, so I think it might be helpful to add specs where we set the constant in specs. There should be examples of how to do that in other specs.